.typos.toml

```diff
@@ -0,0 +1,34 @@
+[default.extend-identifiers]
+AnserStyle = "AnserStyle"
+datas = "datas"
+impl_froms = "impl_froms"
+selfs = "selfs"
+strustS = "strustS"
```

Comment:
This should probably be `structS`, but the whole test hurts my brain:

![image](https://github.com/rust-lang/rust-analyzer/assets/308347/8bd17c67-ed23-4ade-a9d2-3a812ff1a2e6)